tasks/main.yml

```diff
@@ -15,9 +15,18 @@
 - name: Add Docker repository.
   apt_repository: repo='deb https://get.docker.io/ubuntu docker main' state=present
 
-- name: Install packages
-  apt: name="{{ item }}" update_cache=yes
-  with_items: packages
+- name: Install support packages
+  apt: name="{{ item }}"
+  with_items:
+    - wget
+
+- name: Install docker package
+  apt: name="{{ item }}{% if docker_version is defined %}={{ docker_version }}{% endif %}"
+  with_items:
+    - lxc-docker
+
+- name: Do not use sysv to start docker daemon
```

Comment:
The package does provide both:
```
root@docker-server:~# dpkg -S /etc/init.d/docker /etc/init/docker.conf
lxc-docker-1.7.0: /etc/init.d/docker
lxc-docker-1.7.0: /etc/init/docker.conf
```
However I still can't reproduce it after upgrading Ansible from 1.9.0 to 1.9.2